### ISSUE TITLE
[Accessibility Bug] Contrast issue in tagline

### ISSUE THREAD

**Issue by bhavykhatri** [cotes2020/jekyll-theme-chirpy#447]
## Checklist

- [x] I have read the [tutorials](https://chirpy.cotes.info/categories/tutorial/) and know the correct effect of the functional design.
- [x] There are no similar reports on [existing issues](https://github.com/cotes2020/jekyll-theme-chirpy/issues?q=is%3Aissue) (including closed ones).
- [x] I found the bug on the latest code of the `master` branch.

## Describe the bug

Contrast in both dark and light mode for tagline is not according to the guideline.

![image](https://user-images.githubusercontent.com/26280748/142759861-d960be83-6d3e-494a-926e-af22015b455b.png)
![image](https://user-images.githubusercontent.com/26280748/142759881-dc755b6a-2d0b-4fa5-be55-a7a8a907bfdf.png)

I also felt that the tagline has contrast issue and the text is not really that visible as it should be

### To Reproduce

1. Use Accessiblity insight for windows (or any other contrast tool)
2. Go to contrast section.
3. Pick up the color for background and text
4. Notice the ratio is not as per the guideline.

### Expected behavior

Text should be properly visible as per accessibility guideline.

### Screenshots

Provided above

### Software

- Ruby version: 2.7.4p191 
- Gem version:  3.1.6
- Bundler version: 2.2.30

### Additional context

Possible resolution would be to just increase the contrast ratio a bit like this which would resolve the issue.
![image](https://user-images.githubusercontent.com/26280748/142760348-14dace19-367e-4d08-ae4e-38ef6c060a01.png)

**Comment by cotes2020** [comment]
Thank you for submitting this issue, it is very detailed and there is a related PR!

The low color contrast ratio is to prevent the sub-title (tagline) from getting the first attention. There are some other texts on this theme also use this strategy, for example, the metadata of the posts, the copyright information of the footer, etc.
